src/services/rfqm_service.ts

```diff
@@ -759,7 +760,9 @@ export class RfqmService {
             if (!isTxMined) {
                 const newGasPrice = await this._protocolFeeUtils.getGasPriceEstimationOrThrowAsync();
 
-                if (gasPrice.lt(newGasPrice)) {
+                // Geth only allows replacement of transactions if the replacement gas price
+                // is at least 10% higher than the gas price of the  transaction being replaced
+                if (newGasPrice.gte(gasPrice.multipliedBy(MIN_GAS_PRICE_INCREASE + 1))) {
```

Comment:
Can we add a unit test for this in `rfqm_service_test.ts`? (No need to retroactively add unit tests for other cases in `completeSubmissionLifecycleAsync`)

A test like this will likely need to "capture" the arguments passed to `blockchainUtils` - reference: https://www.npmjs.com/package/ts-mockito#capturing-method-arguments

Once the arguments are captured, you can assert that the `options.gasPrice` is what you expect.